### PR TITLE
boveri Molecular_test node updates

### PR DESCRIPTION
**Pull request by ganjigajanan**

### CODE REVIEW

File: gdcdictionary/schemas/molecular_test.yaml [hunk 1]
```diff
@@ -6940,7 +6980,26 @@ properties:
...
+      - U/L
+      - ukat/L
+      - umol/L
+      - x10^3 cells/mcL
+      - x10^9 cells/L
+      - Not Report
+      - Unknown
+      - count x10^9 cells/L
+    deprecated_enum:
+      - count x10^9 cells/L
```
Comment (wteouchicago) [inline gdcdictionary/schemas/molecular_test.yaml:7002]: DICT-95 also lists x10^3/mcL under deprecated

File: gdcdictionary/schemas/molecular_test.yaml [hunk 2]
```diff
@@ -443,6 +454,11 @@ properties:
       - "_terms.yaml#/cytoband/common"
     type: string
 
+  days_to_test:
+    $ref:
+      - "_terms.yaml#/days_to_test/common"
+    type: integer
+
```
Comment (wteouchicago) [inline gdcdictionary/schemas/molecular_test.yaml:461]: seems like we can do some validation here for min/max, e.g. e.g. -32872 and 32872

File: gdcdictionary/schemas/molecular_test.yaml [hunk 3]
```diff
@@ -6940,7 +6980,26 @@ properties:
...
+      - mmol/L
+      - mU/mol Hb
+      - U/g Hb
+      - U/mL
+      - U/L
+      - ukat/L
+      - umol/L
+      - x10^3 cells/mcL
+      - x10^9 cells/L
+      - Not Report
```
Comment (wteouchicago) [inline gdcdictionary/schemas/molecular_test.yaml]: Jira says "Not Report", but I'm guessing it should be "Not Reported"

File: gdcdictionary/schemas/molecular_test.yaml (hunk 1 above)
Comment (wwysoc2) [reply gdcdictionary/schemas/molecular_test.yaml:7002]: mg/dl <- Should this be "mg/dL" to match with the other enums?

Comment (wwysoc2) [reply gdcdictionary/schemas/molecular_test.yaml:7002]: Also "x10^9/L" is under the deprecated value.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: gdcdictionary/schemas/molecular_test.yaml (hunk 2 above)
Comment (ganjigajanan) [reply gdcdictionary/schemas/molecular_test.yaml:461]: updated

File: gdcdictionary/schemas/molecular_test.yaml (hunk 1 above)
Comment (ganjigajanan) [reply gdcdictionary/schemas/molecular_test.yaml:7002]: changed mg/dl to mg/dL 
x10^3/mcL & x10^9/L were added to ticket by mistake, removed from ticket

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**